### PR TITLE
GCW-2366 logistics tab : Add rows

### PR DESCRIPTION
**Pull request by patrixr**
Adds extra lines of data to the logistic tabs. Those are not yet editable.

From the design :
screenshot 2019-02-14 at 11 40 42 am

### CODE REVIEW

File: app/controllers/orders/order_types.js [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
 
-export default detail.extend();
+export default detail.extend({
+
+  logisticDataRows: Ember.computed('model', function () {
+    const order = this.get('model');
+    return [
+      // Schedule
+      {
+        label: this.get('i18n').t('order_details.logistics.scheduled'),
```
Comment (Darshan-Chauhan) [inline app/controllers/orders/order_types.js]: This seems like repetition of same... Can we create a function which will take params and returns hash depending on param??

Comment (patrixr) [reply app/controllers/orders/order_types.js]: we can, but it'd end up similar, calling the function again and again.

e.g
```
makeRow(
        this.get('i18n').t('order_details.logistics.type'),
        order.get('bookingTypeLabel'),
        function action() {},
        'tv',
        'type'
)
```

I'm okay with that, but i don't find it significantly better.
It's just data after all.
We could move everything to the `hbs` and not have a foreach, but i think it clutters the template too much, this is long but very understandable

Comment (patrixr) [reply app/controllers/orders/order_types.js]: I could make separate functions, e.g `makeVehicleRow()` , `makeScheduleRow()`, and return 
```
[ makeVehicleRow(), makeScheduleRow() ]
```

but then again, is it really a better alternative, i'm not so sure tbh

**Comment by Darshan-Chauhan** [comment]
@patrixr , don't merge anything to master yet.
I'm working on fix for language-switcher which we want to release. 
Once the fix is released then feel free to merge. :)

**Comment by patrixr** [comment]
> @patrixr , don't merge anything to master yet.
> I'm working on fix for language-switcher which we want to release.
> Once the fix is released then feel free to merge. :)

Got it :)

**Comment by Darshan-Chauhan** [comment]
@patrixr , feel free to merge now :)

**Pull request merged**